sdk/servicebus/service-bus/src/util/sasServiceClientCredentials.ts

```diff
@@ -17,13 +17,11 @@ export class SasServiceClientCredentials implements ServiceClientCredentials {
    */
   constructor(sharedAccessKeyName: string, sharedAccessKey: string) {
     if (typeof sharedAccessKeyName !== "string") {
-      throw new Error(
-        "sharedAccessKeyName cannot be null or undefined and must be of type string."
-      );
+      throw new Error("Missing SharedAccessKeyName in Connection String.");
     }
 
     if (typeof sharedAccessKey !== "string") {
-      throw new Error("sharedAccessKey cannot be null or undefined and must be of type string.");
+      throw new Error("Missing SharedAccessKey in Connection String.");
```

Comment:
Yeah it gives a `Error: 401 - Generic: . ` error message, which doesn't seem helpful enough

Reply:
401 is for Unauthorized Error, so that should be good enough. We can skip these checks.
We dont make these checks on the connection string for the `ServiceBusClient`.

Reply:
Updated